packages/frontend/components/composed/community/ResearchUnitsList/ResearchUnitsList.styles.tsx

```diff
@@ -0,0 +1,94 @@
+import styled from "styled-components";
+import { pxToRem } from "@wdp/lib/theme/functions";
+import BaseImage from "next/image";
+import { respond, lGrid } from "theme/mixins";
+
+export const Inner = styled.div`
+  ${lGrid()}
+  padding-block-start: ${pxToRem(100)};
+  padding-block-end: ${pxToRem(100)};
+
+  ${respond(
+    `
+      grid-template-columns: 1fr;
+      grid-template-rows: auto;
+      grid-template-areas:
+        "text"
+        "image";
+    `,
+    70
+  )}
+`;
+
+export const TextColumn = styled.div`
+  grid-column: 2 / 6;
+
+  ${respond(
+    `
+      grid-area: text;
+    `,
+    70
+  )}
+`;
+
+export const HeaderBlock = styled.div`
+  padding-block-end: ${pxToRem(12)};
+  border-bottom: 1px solid currentColor;
+`;
+
+export const UnitsList = styled.ul`
+  padding-block-start: ${pxToRem(28)};
+  padding-block-end: var(--padding-lg);
+  border-bottom: 1px solid currentColor;
+`;
+
+export const ListItem = styled.li`
+  & + & {
+    margin-block-start: ${pxToRem(20)};
+  }
+`;
+
+export const UnitLink = styled.a`
+  display: flex;
+
+  &:hover,
+  &:focus {
+    text-decoration: underline;
+  }
```

Comment:
Is this right?

Reply:
The link hover styles are a fancy box-shadow - if you use the `a-link` class, it will apply all the styles you need. I also suggest using the ArrowList component for the list